docs/develop/understanding-the-javascript-api-for-office.md

```diff
@@ -64,15 +124,28 @@ Office.initialize = function (reason) {
     });
  };
 ```
-For more information, see [Office.initialize Event](https://dev.office.com/reference/add-ins/shared/office.initialize) and [InitializationReason Enumeration](https://dev.office.com/reference/add-ins/shared/initializationreason-enumeration). 
+
+For more information, see [Office.initialize Event](https://dev.office.com/reference/add-ins/shared/office.initialize) and [InitializationReason Enumeration](https://dev.office.com/reference/add-ins/shared/initializationreason-enumeration).
+
+### Major differences between Office.initialize and Office.onReady()
+
+- You can assign only one handler to `Office.initialize` and it is called, only once, by the Office infrastructure. But you can call `Office.onReady()` in different places in your code and use different callbacks. For example, your code could call `Office.onReady()` as soon as your custom script loads (in, for example, an Immediately Invoked Function Expression) with a callback that runs initialization logic. But you could also have a button in the task pane, whose script calls `Office.onReady()` with a different callback. If so, the second callback runs when the button is clicked.
```

Comment:
Nit: you have two sentences starting with "but" here, which makes this read a little confusing.